configurationdemo/source/cpp/CMakeLists.txt

```diff
@@ -14,70 +14,22 @@
 #  limitations under the License.
 #
 
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 2.8.12)
+project(ConfigurationDemo CXX)
 
-project (Cpp-SDK-sample)
-enable_language(CXX)
+# Disable unused features
+set(KAA_WITHOUT_NOTIFICATION 1)
```

Comment:
please, decide between 1/0 , ON/OFF and 
set(KAA_WITHOUT_NOTIFICATION 1) / set(WITH_EXTENSION_NOTIFICATION OFF)